Source/Extensions/UIViewExtensions.swift

```diff
@@ -1,6 +1,14 @@
 import Foundation
 import UIKit
 
+@objc protocol Updatable {
+    func update(with style: Any?)
+}
+
+extension UIView: Updatable {
+    func update(with style: Any?) {}
```

Comment:
Why do we need this protocol? Why can't the concrete object just hold the method and we want it accessed via some protocol interface?